test/parallel/test-dgram-send-default-host.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+if (process.platform.indexOf('win') >= 0)
```

Comment:
`common.isWindows` is how this is done in tests, it should also output that it's being skipped, if you grep for 'Skipped' and 'Windows' you'll find examples of this, like this from test/parallel/test-cluster-bind-privileged-port.js:

```
if (common.isWindows) {
  console.log('1..0 # Skipped: not reliable on Windows.');
  return;
}
```